portfolio/src/main/webapp/styles/style.css

```diff
@@ -1,35 +1,36 @@
 /* General Elements */
 body {
-  color:rgb(160, 160, 160);
-  background-color: #02003C;
+  color:white;
+  background-color: #41485B;
```

Comment:
order.

Reply:
See #12 . But I am confused why we do this here as it is not alphabetical?